examples/lib/stories/animations/basic.dart

```diff
@@ -64,7 +63,7 @@ class BasicAnimations extends BaseGame with TapDetector {
   }
 
   @override
-  void onTapDown(TapDownDetails evt) {
-    addAnimation(Vector2(evt.globalPosition.dx, evt.globalPosition.dy));
+  void onTapDown(TapDownInfo evt) {
```

Comment:
how about calling it *Event?

Reply:
That was my first idea, but then it will clash with some of the mouse event classes from flutter. So I choose Info which don't clash with any flutter class.

I am not 100% happy with that name but couldn't find anything better, totally open to suggestions

Reply:
ah I thought flutter used Details, that is why I suggested Event. in that case, Info is fine by me